optimum/utils/normalized_config.py

```diff
@@ -94,6 +94,30 @@ def __getattr__(self, attr_name):
         return super().__getattr__(attr_name)
 
 
+class NormalizedEncoderDecoderConfig(NormalizedSeq2SeqConfig, NormalizedVisionConfig):
```

Comment:
Can this be refactored with `NormalizedTextAndVisionConfig`?

Also, what are `ENCODER_CONFIG` and `DECODER_CONFIG`?

Reply:
Probably not as the use of both the configs is different. Though built in similar way, the new normalised config is to be used with any type of `EncoderDecoder` type models text, vision, speech etc. The idea is to not force any normalised config beforehand and it takes the configs from the encoder and decoder part of the model while initialising the onnx config. Removed the incorrect inheritance to avoid confusion.

I have removed the `ENCODER_CONFIG` and `DECODER_CONFIG` since they are not used anymore.